packages/@rnw-scripts/metro-dev-config/metro.dev.config.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ * @ts-check
+ */
+
+const {mergeConfig} = require('metro-config');
+
+const MetroConfig = require('@rnx-kit/metro-config');
+const {MetroSerializer} = require('@rnx-kit/metro-serializer');
+const {
+  DuplicateDependencies,
+} = require('@rnx-kit/metro-plugin-duplicates-checker');
+
+function makeMetroConfig(customConfig = {}) {
+  return mergeConfig(MetroConfig.makeMetroConfig(customConfig), {
+    resolver: {
+      blockList: MetroConfig.exclusionList([
+        // This prevents "react-native run-windows" from hitting: EBUSY: resource
+        // busy or locked for files produced by MSBuild
+        /.*\/build\/.*/,
+        /.*\/target\/.*/,
+        /.*\.tlog/,
+      ]),
+    },
+    serializer: {
+      customSerializer: MetroSerializer([
+        DuplicateDependencies({
+          // Duplicate dependencies introduced by external code
+          ignoredModules: ['react-is'],
```

Comment:
Would love to get a sense of if this can be fixed.  I've been hitting this internally too, -- seems like some dependency somewhere needs updating?

Reply:
It looked like it was coming in via a prop-types package, which there is work to remove from RN. So it may end up being resolved soon.

Reply:
can confirm, we're seeing commits in main to remove prop-types